ui/app/pages/swap/swap-content/swap-amount-row/swap-amount-row.component.js

```diff
@@ -105,40 +104,22 @@ export default class SwapAmountRow extends Component {
     this.updateAmount(newAmount)
   }
 
-  renderInput () {
-    const { amount, inError, swapFromToken } = this.props
+  render () {
+    const { amount, swapFromToken } = this.props
 
     return swapFromToken ?
       (
         <UserPreferencedTokenInput
-          error={inError}
```

Comment:
**Note to reviewer:**

Removed it because the red border, in case of an error, was looking quite ugly. We still have the error message just below.